CHANGELOG.md

```diff
@@ -18,9 +18,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - The `async-operations` endpoint will now omit `output` instead
       of returning `none` when the operation did not return output.
   - **CUMULUS-2309**
-    - Removed `@cumulus/api/models/granule.unpublishAndDeleteGranule` in favor of `@cumulus/api/lib/granule-remove-from-cmr.unpublishGranule` and `@cumulus/api/lib/granule-delete.deleteGranuleAndFiles`.
+    - Removed `@cumulus/api/models/granule.unpublishAndDeleteGranule` in favor
```

Comment:
Changes here were almost entirely the result of an editor re-wrap on accepting changes.  I can revert if that's obnoxious, however we *did* agree it was good practice at one point.

Reply:
Test comment

Reply:
Isn't it going to result in a conflict when merging back to master?

Reply:
These are all in the RDS feature list, so.... I don't think so?

Reply:
_expands_, Oh. Fair enough!